src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,49 +1,114 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final double INCREMENT_BY_HALF = 1.5;
+    private T[] values;
+    private int size;
+
+    public ArrayList() {
+        this.values = (T[]) new Object[DEFAULT_CAPACITY];
+        this.size = 0;
+    }
+
+    private void resizing() {
```

Comment:
```suggestion
    private void resize() {
```
let’s move private methods to the bottom of the file, or at least after the place where we call them

Reply:
done